.github/workflows/integration.yml

```diff
@@ -2,7 +2,7 @@ name: neow3j-integration
 
 on:
   pull_request:
-    types: [ assigned, review_requested, ready_for_review ]
+    types: [ review_requested, ready_for_review ]
```

Comment:
Why did you remove this?

It's an easy way to re-trigger the integration test if there's any error. I would keep it. I remember when @csmuller and I decided to do this, because there was no easy way to retrigger only the int tests...

Reply:
You can just as easily convert it to `draft` and mark it as `ready for review` again.

Reply:
And hence not trigger it three times whenever you open a new PR.